### PR TITLE
Enforce some PEP8 formatting before build (backport to maint-3.9)

### PR DESCRIPTION
**Pull request by mormj**
Uses a github action to enforce PEP8 with some exceptions
Reformat all the python code to get it to pass this check

Backport of #5308  (would let it sit on master a bit before merging)

### CODE REVIEW

File: gnuradio-runtime/python/gnuradio/gr/CMakeLists.txt [hunk 1]
```diff
@@ -27,7 +27,6 @@ if(ENABLE_TESTING)
   set(GR_TEST_TARGET_DEPS "")
   set(GR_TEST_LIBRARY_DIRS "")
   set(GR_TEST_PYTHON_DIRS
-    ${CMAKE_BINARY_DIR}/gruel/src/python
```
Comment (marcusmueller) [inline gnuradio-runtime/python/gnuradio/gr/CMakeLists.txt:30]: loool. yeah, that shouldn't be there. ooops...

**Comment by marcusmueller** [comment]
Can we also get that check / corrector as script somewhere in-tree? That way, we could tell contributors that, hey, please run this and that on your changes.

**Comment by mormj** [comment]
> Can we also get that check / corrector as script somewhere in-tree? That way, we could tell contributors that, hey, please run this and that on your changes.

Good point - just added a script in the tools/ directory that does both the modification and check

File: grc/main.py [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
 #
 
-import argparse, logging, sys
+from gi.repository import Gtk
+import argparse
+import logging
+import sys
 
 import gi
 gi.require_version('Gtk', '3.0')
```
Comment (willcode) [inline grc/main.py:13]: `gi.require_version()` should be before `from gi.repository ...`

**Review by willcode — CHANGES_REQUESTED** [review]
Over-pepped some ordering by putting imports before some library load instruction. Might not be noticed if we're calling for the system default, but could fail on some systems.

**Pull request merged**